### DISCUSSION TITLE
Type-graphql with nextjs

### DISCUSSION THREAD

**Discussion by edw19** [MichalLytek/type-graphql#933] [Q&A] [answered]
I am trying to use type-graphlq with nextjs but I get that error, how could I fix it

![](https://cdn.discordapp.com/attachments/752647196419031042/857822613094662154/unknown.png)

![](https://cdn.discordapp.com/attachments/752647196419031042/857820483150938112/unknown.png)

**Comment by MichalLytek** [comment, marked as answer]
Please read #55
